src/actions/browser_action.cr

```diff
@@ -1,5 +1,11 @@
+class Fiber
+  property breeze_request : BreezeRequest?
+end
+
 abstract class BrowserAction < Lucky::Action
+  before store_breeze_request
```

Comment:
This would likely be added to `Lucky::Action` automatically through monkey patching when Breeze is required. That way user doesn't have to do anything